docs/release/trg-10/trg-10-01.md

```diff
@@ -0,0 +1,123 @@
+
+- Are themselves test code (no "test of tests").
+- Are autogenerated code, for example Swagger-generated API clients.
+- Are configuration files with no logic to test.
+- Are experimental or prototype code, for example incomplete implementations hidden behind feature toggles.
+- Are boilerplate code or entity classes that follow a known, predictable pattern.
+- Are code for logging, metrics, or monitoring.
+- Depend on platform-specifics, for example hardware that cannot be simulated in a test.
+
+All exceptions should be documented and approved by the project's lead/main committers.
+
+### 2. Analysis and Reporting
+
+2.1. **Tools for Code Coverage Reporting**
+
+- The recommended code coverage reporting tool is **SonarQube Cloud** as provided by Elipse Tractus-X at this [reference link](https://sonarcloud.io/organizations/eclipse-tractusx/projects).
+- Please note that SonarQube Cloud does not calculate the code coverage itself. To include coverage results you must set up a third-party coverage analysis tool and configure SonarQube Cloud to import the analysis results produced by that tool.
+- SonarQube Cloud supports the import of coverage data in formats native to a variety of tools for a variety of languages. It also provides a possibility to import a generic format from tools that are not directly supported. See the guideline in section [4. Resources and Examples](#sonarqube-implementation-guides) as reference on how to implement this.
+- The tool or tools used for test coverage analysis and to produce the code coverage report should be specified and documented within the respective Eclipse Tractus-X project.
+
+2.2. **Regular Review**
+
+- Code coverage should be measured by all Eclipse Tractus-X software products at least once before each release.
+- Coverage measurement results should be integrated into CI/CD pipelines and automated as part of the release process.
+
+### 3. Quality Assurance
+
+3.1. **Code Review Requirements**
+
+- Reviewers must ensure that new or modified code sections are sufficiently covered by test cases.
+- Code changes that lower the overall coverage below the defined Quality Gate threshold may only be accepted in exceptional cases.
+
+3.2. **Risk Analysis**
+
+- If the code coverage value for an Eclipse Tractus-X project falls below 80.0% for an (upcoming) release and this cannot be improved anymore in time before the release, this potential risk should be documented and approved by the project lead/main committers.
+
+### 4. Resources and Examples
+
+<a id="sonarqube-implementation-guides"></a>
+
+#### 4.1 **SonarQube Cloud Implementation Guides**
+
+See the following SonarQube Cloud guideline for the importing of test coverage reports with detailed guides for several popular programming languages as [reference](https://docs.sonarsource.com/sonarqube-cloud/enriching/test-coverage/overview/).
+
+#### 4.2. **GitHub Workflow Integration Code Example**
```

Comment:
the TRG is about test coverage and measuring that with sonar, as far as I understood, and the GH workflow example for CI integration is about executing and publishing unit and integration tests, that doesn't make sense to me